test/test.js

```diff
@@ -30,6 +37,9 @@ suite('GaiaButton', function() {
     this.el.setAttribute('disabled', true);
     assert.equal(this.el.getAttribute('disabled'), 'true');
     assert.equal(this.el.disabled, true);
+
+    this.el.toggled = true;
+    assert.equal(this.el.getAttribute('toggled'), '');
```

Comment:
I don't see how the above lines are related to 'disabled' behaviour. Please move to a separate test to test that the `toggled` attribute can be get and set in various ways.